src/App.jsx

```diff
@@ -0,0 +1,22 @@
+import './App.css';
+import { Routes, Route } from 'react-router-dom'
+import Footer from './components/Footer';
+import Navbar from './components/Navbar';
+import Home from './components/Home';
+import About from './components/About';
+
+function App() {
+  return (
+    <div>
+      <Navbar />
+      <Footer />
+
+      <Routes>
+        <Route path='/home' element={<Home />} />
```

Comment:
it would be better if you make the home route just like "/"

Reply:
Ok :)